task_handler_layer/task_buoy/msg/ResizedImage.msg

```diff
@@ -0,0 +1,3 @@
+uint32 original_height
```

Comment:
Its already there https://github.com/auv-iitk/auvwiki/wiki/faq#how-to-remove-lint-errors